index.html

```diff
@@ -0,0 +1,93 @@
+    <title>Aprendiendo Flexbox</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <header>
+        <img src="images/Imagen2.png" alt="logo" width="111px" height="74px"> 
+        <div class="boton">≡</div>
+    </header>
+    <main>
+        <h1 class="caja miNombre">
+            Felipe <br> Haybar 
+        </h1>
+        <section class="saludo">
+            <p>
+            Hello! I am a software developer! I can help you build a product, feature or website. Taka a look of my works. If you like what you see and have a project you need coded, don’t hesitate and contact me.
+            </p>
+         </section>    
+    </main>
+    <section class="collabBoton"><p> start colaboration </p></section>
+    <div class="art"><img src="images/art.png" alt="artistic forms" width="320px"></div>
+    <section class="projects">
+        <h4>Projects</h4>
+        <img src="images/project1.png" alt="project1" width="320px">
+        <div class="pData">
+            <h2>Project name goes here</h2>
+            <ul>
+                <li>HTML/CSS</li>
+                <li>Ruby on Rails</li>
+                <li>Java script</li>
+            </ul>
+            <h3>See this project →</h3>
+        </div>
+        <img src="images/project2.png" alt="project2" width="320px">
+        <div class="pData">
+            <h2>Project name goes here</h2>
+            <ul>
+                <li>HTML/CSS</li>
+                <li>Ruby on Rails</li>
+                <li>Java script</li>
+            </ul>
+            <h3>See this project →</h3>
+        </div>
+        <img src="images/project3.png" alt="project3" width="320px">
+        <div class="pData">
+            <h2>Project name goes here</h2>
+            <ul>
+                <li>HTML/CSS</li>
+                <li>Ruby on Rails</li>
+                <li>Java script</li>
+            </ul>
+            <h3>See this project →</h3>
+        </div>
+        <img src="images/project4.png" alt="project4" width="320px">
+        <div class="pData">
+            <h2>Project name goes here</h2>
+            <ul>
+                <li>HTML/CSS</li>
+                <li>Ruby on Rails</li>
+                <li>Java script</li>
+            </ul>
+            <h3>See this project →</h3>
+        </div>
+        <img src="images/project 4.png" alt="project5" width="320px">
+        <div class="pData">
+            <h2> Project name goes here</h2>
+            <ul>
+                <li>HTML/CSS</li>
+                <li>Ruby on Rails</li>
+                <li>Java script</li>
+            </ul>
+            <h3>See this project →</h3>
+        </div>
+    </section>
+    <h4>About</h4>
+    <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ea eiusmod</p>
+    <section class="collabBoton"><p> Get my CV &nbsp;&nbsp; <u>↓</u>   </p></section>
```

Comment:
- [ ] Kindly use a link `a` tag or `button` element here as well. This will allow the users to be linked to another page. 👍 
- [ ] You can also wrap it with a `div `not `section`. `section `tag is best used for grouping similar contents. 😄